### PR TITLE
Allow deploy without grisp runtime and fix version warning

### PR DESCRIPTION
**Pull request by sylane**

### CODE REVIEW

File: src/rebar3_grisp_util.erl [hunk 1]
```diff
@@ -60,6 +61,13 @@ otp_build_root(State, Version) ->
 otp_install_root(State, Version) ->
     filename:join([root(State), "otp", Version, "install"]).
 
+otp_install_release(InstallRoot) ->
```
Comment (eproxus) [inline src/rebar3_grisp_util.erl]: Maybe rename this to `otp_install_release_subdir`? Sounds like that function is actually going to install a release 😄

Comment (sylane) [reply src/rebar3_grisp_util.erl]: ok

Comment (sylane) [reply src/rebar3_grisp_util.erl]: It is not realy the subdir, it return the version itself, would `otp_install_release_version` be ok ?

Comment (eproxus) [reply src/rebar3_grisp_util.erl]: Sure, that would work. My problem with it is that the function name is a verb, when it should be a noun. That would fix it.

**Pull request merged**